documentation/modules/ROOT/pages/connectors/mysql.adoc

```diff
@@ -2120,12 +2121,14 @@ Information about the properties is organized as follows:
 
 * xref:mysql-required-connector-configuration-properties[Required connector configuration properties]
 * xref:mysql-advanced-connector-configuration-properties[Advanced connector configuration properties]
-* xref:mysql-pass-through-configuration-properties[Pass-through configuration properties]
+* xref:debezium-{context}-connector-database-history-and-pass-through-configuration-properties[{prodname} connector database history and pass-through configuration properties]
```

Comment:
See my other comment below; DB drive pass-through properties should be their own section, on the same level as the history ones.